Objects/dictobject.c

```diff
@@ -5553,11 +5594,20 @@ _PyObject_StoreInstanceAttribute(PyObject *obj, PyDictValues *values,
     assert(values != NULL);
     assert(Py_TYPE(obj)->tp_flags & Py_TPFLAGS_MANAGED_DICT);
 
-    if(!Py_CHECKWRITE(obj)){
+    if (!Py_CHECKWRITE(obj)){
         PyErr_WriteToImmutable(obj);
         return -1;
     }
 
+    //TODO: PYRONA: The addition of the key is complex here.
```

Comment:
Good with the labelling. We should go through all our phase3 code and make sure our TODOs and FIXMEs are all labelled thus.